sdk/src/TokenClient.cs

```diff
@@ -652,18 +682,16 @@ public PagedBanks GetBanksBlocking(string search)
         }
 
         /// <summary>
-        /// Parse the token request callback URL to extract the state and the token ID. Verify that the
+        /// Parse the token request callback Parameter to extract the state and the token ID. Verify that the
         /// state contains the CSRF token hash and that the signature on the state and CSRF token is
         /// valid.
         /// </summary>
-        /// <param name="callbackUrl">the token request callback url</param>
+        /// <param name="callbackParams">the token request callback parameters</param>
         /// <param name="csrfToken">the csrf token</param>
         /// <returns>an instance of <see cref="TokenRequestCallback"/></returns>
-        public TokenRequestCallback ParseTokenRequestCallbackUrlBlocking(
-            string callbackUrl,
-            string csrfToken = "")
+        public TokenRequestCallback ParseTokenRequestCallbackParamsBlocking(string callbackParams, string csrfToken)
```

Comment:
The data type in Java SDK for callbackParams is `Map<String, String>`. In Javascript SDK it expects an object with these params `{tokenId: string, signature: string, state: string}`. It seems wrong to expect a string in C# SDK.